Sources/Controllers/Map/OAMapActions.h

```diff
@@ -8,6 +8,7 @@
 
 #import <Foundation/Foundation.h>
 #import <CoreLocation/CoreLocation.h>
+#import "OAApplicationMode.h"
```

Comment:
If you do not need to use the implementation of a class in a header file, you should specify ```@class <class name>;``` instead and remove the import